lockable/lockable.py

```diff
@@ -83,7 +75,21 @@ def parse_requirements(requirements_str: (str or dict)) -> dict:
             elif value.lower() == "false":
                 value = False
             requirements[key] = value
-        return requirements
+        return unflatten(requirements)
+
+    @staticmethod
+    def parse_requirements(requirements_str: (str or dict)) -> dict:
+        """ Parse requirements """
+        if not requirements_str:
+            return {}
+        if isinstance(requirements_str, dict):
+            return requirements_str
+        try:
+            return json.loads(requirements_str)
+        except json.decoder.JSONDecodeError as error:
+            if error.colno > 1:
```

Comment:
What does this mean in practice? Add comment perhaps?

Reply:
this was original way to detect if string is not json format. there is at least one bug here actually - should verify first char, not colno... will fix it in another PR

Reply:
added comment anyway..

Reply:
fixed here: https://github.com/jupe/py-lockable/pull/199